Makefile

```diff
@@ -342,7 +342,7 @@ e2etest:
 playwright:
 	cd e2e_playwright; \
 	rm -rf ./test-results; \
-	pytest --browser webkit --browser chromium --browser firefox --video retain-on-failure --screenshot only-on-failure --output ./test-results/ -n auto --reruns 1 --reruns-delay 1 --rerun-except "Missing snapshot" -r aR -v
+	pytest --browser webkit --browser chromium --browser firefox --video retain-on-failure --screenshot only-on-failure --output ./test-results/ -n auto --reruns 1 --reruns-delay 1 --rerun-except "Missing snapshot" --durations=5 -r aR -v
```

Comment:
Is there a reason why we want to show only the 5 slowest?

Reply:
Not really, but we need to specify any reasonably small number here since. Otherwise, it would make looking through the logs a bit harder (we have over 1k test cases):

<img width="952" alt="image" src="https://github.com/streamlit/streamlit/assets/2852129/ae4f4e8c-18c3-4a2e-90f9-ef9361919dbb">

we could also go with the top 10 or something probably < 100.

Reply:
And this is mainly for having some easy checks that there isn't a test doing something horribly wrong (e.g. requiring 10 minutes). As long as everything stays in <1 min, its fine

Reply:
I will keep this at 5 for now since I think its mostly useful for outlier detection.